.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   release-package:
     environment: publish
-    runs-on: windows-latest
+    runs-on: ubuntu-latest
     permissions:
       contents: write # for googleapis/release-please-action to create release commit
       pull-requests: write # for googleapis/release-please-action to create release PR
```

Comment:
I think you can remove `packages` here since that's only needed in the next job.

Also, I think you need `issues: write` in the latest version of release please, as we have in Java: https://github.com/open-feature/java-sdk-contrib/blob/main/.github/workflows/release-please.yml#L12